src/CommandHandler.cpp

```diff
@@ -395,6 +395,19 @@ void CommandHandler::handle(std::string const &Command) {
     LOG(Sev::Error, "Can not parse json command: {}", Command);
     return;
   }
+
+  if (auto x = find<std::string>("service_id", Doc)) {
+    if (x.inner() != Config.service_id) {
+      return;
```

Comment:
I think we should log when this happens, it'll make the circumstance of using an invalid `service_id` easier to debug than if all the file writers just silently ignore the command.

Reply:
I make it `Debug` level though, because I assume that this may happen all the time under normal operations if writers share a topic.